src/http/api/account.ts

```diff
@@ -42,57 +52,52 @@ export function createGetAccountHandler(accountService: AccountService) {
     return async function handleGetAccount(ctx: AppContext) {
         const logger = ctx.get('logger');
         const site = ctx.get('site');
+        let account: Account | null = null;
+        const db = ctx.get('db');
 
-        // Validate input
-        const handle = ctx.req.param('handle') || '';
+        const apCtx = fedify.createContext(ctx.req.raw as Request, {
+            db,
+            globaldb: ctx.get('globaldb'),
+            logger,
+        });
 
-        if (handle === '') {
-            return new Response(null, { status: 400 });
+        const defaultAccount = await accountRepository.getBySite(
+            ctx.get('site'),
+        );
+
+        const handle = ctx.req.param('handle');
+        if (handle === 'index') {
```

Comment:
Hm, if would be nice if we didn't have to hardcode this. What about assuming if the param is empty that the request is for the current user?